### PR TITLE
add swift-models quickstart

### PR DESCRIPTION
**Pull request by brettkoonce**

### CODE REVIEW

File: Installation.md [hunk 1]
```diff
@@ -390,6 +390,13 @@ var x = Tensor<Float>([[1, 2], [3, 4]])
 print(x + x)
 ```
 
+## Run [swift-models](https://github.com/tensorflow/swift-models)
+
+```swift
```
Comment (saeta) [inline Installation.md:395]:
```suggestion
```sh
```

**Review by saeta — COMMENTED** [review]
@dabrahams to make further suggestions.

File: Installation.md [hunk 2]
```diff
@@ -390,6 +390,13 @@ var x = Tensor<Float>([[1, 2], [3, 4]])
...
+git clone https://github.com/tensorflow/swift-models.git
+cd swift-models
+swift run
+```
```
Comment (dabrahams) [inline Installation.md:400]:
```suggestion

Swift will print an error with a list of executable names that exercise different models.  Issue `swift run` *executable-name* to select the model you're interested in.
```

File: Installation.md [hunk 3]
```diff
@@ -390,7 +390,15 @@ var x = Tensor<Float>([[1, 2], [3, 4]])
...
 
+## Run [swift-models](https://github.com/tensorflow/swift-models)
 
+```sh
+git clone https://github.com/tensorflow/swift-models.git
+cd swift-models
+swift run
+```
+
+Swift will print an error with a list of executable names that exercise different models.  Issue `swift run` *executable-name* to select the model you're interested in.
```
Comment (dabrahams) [inline Installation.md:401]:
```suggestion
Swift will print an error with a list of executable names that exercise different models.
Issue `swift run` *executable-name* to select the model you're interested in.
```

File: Installation.md [hunk 4]
```diff
@@ -390,7 +390,16 @@ var x = Tensor<Float>([[1, 2], [3, 4]])
...
+## Run [swift-models](https://github.com/tensorflow/swift-models)
 
+```sh
+git clone https://github.com/tensorflow/swift-models.git
+cd swift-models
+swift run
+```
+
+Swift will print an error with a list of executable names that exercise different models.
+Issue `swift run` *executable-name* to select the model you're interested in.
```
Comment (dabrahams) [inline Installation.md:402]:
```suggestion
Issue `swift run` *executable-name* to select the model you're interested in.

```

**Review by dabrahams — APPROVED** [review]

**Pull request merged**